src/werkzeug/serving.py

```diff
@@ -909,7 +917,8 @@ def main():
     import optparse
     from .utils import import_string
 
-    parser = optparse.OptionParser(usage="Usage: %prog [options] app_module:app_object")
```

Comment:
Unrelated, but we should move this to argparse (I'll open an issue for it) since `optparse` is deprecated since Python 2.7.

Reply:
I've opened a separate PR (#1828) to fix this.